### PR TITLE
Update osgeo-gdal.rb

### PR DESCRIPTION
**Pull request by fjperini**
Solution for the problem reported in #1087 #1085 #1086

### CODE REVIEW

File: Formula/osgeo-gdal.rb [hunk 1]
```diff
@@ -142,7 +142,7 @@ def configure_args
       "--with-qhull=internal",
       "--with-libz=#{Formula["libzip"].opt_prefix}",
       "--with-png=#{Formula["libpng"].opt_prefix}",
-      "--with-libtiff=#{Formula["libtiff"].opt_prefix}",
+      "--with-libtiff=internal", # #{Formula["libtiff"].opt_prefix}
       "--with-geotiff=#{Formula["osgeo-libgeotiff"].opt_prefix}",
```
Comment (vincentsarago) [inline Formula/osgeo-gdal.rb]: @fjperini I've tried similar thing yesterday but then I had problem with TiffTags. Using external libgeotiff and internal libtiff might create some side effects

Comment (fjperini) [reply Formula/osgeo-gdal.rb]: @vincentsarago can you report the errors? suggestions?

Comment (vincentsarago) [reply Formula/osgeo-gdal.rb]: Well basically with this configuration I couldn't read any GeoTIFF thus I removed `--with-geotiff` and it seemed to fix it

Comment (vincentsarago) [reply Formula/osgeo-gdal.rb]: I guess it's better to use both libtiff and libgeotiff shipped with GDAL to avoid conflict

Comment (vincentsarago) [reply Formula/osgeo-gdal.rb]: maybe @palmerj could validate this

**Comment by vincentsarago** [comment]
@fjperini another comment not related but could we remove `jasper` ? Because we ship gdal with openjpeg I don't think we need jasper to read jpeg2000 files. 
**I'll need to check** but I though jasper driver will be found first before OpenJPEG thus you need to add `GDAL_SKIP="Jasper"` to force GDAL to use OpenJPEG.

File: Formula/osgeo-gdal.rb (hunk 1 above)
Comment (fjperini) [reply Formula/osgeo-gdal.rb]: yes, maybe we need to use "internal" in both.

**Comment by fjperini** [comment]
@vincentsarago will disable jasper

**Comment by fjperini** [comment]
proj5 #1090

**Comment by vincentsarago** [comment]
👏 thanks for the changed @fjperini

**Comment by fjperini** [comment]
I will merge this

File: Formula/osgeo-gdal.rb (hunk 1 above)
Comment (palmerj) [reply Formula/osgeo-gdal.rb]: Thanks both. I've not had any issues using external geotiff and internal libtiff, but I haven't done wide testing. Seem logical to use internal for both.

**Comment by palmerj** [comment]
Thank you very much

**Pull request merged**